packages/extension/src/ui/features/actions/ApproveTransactionScreen.tsx

```diff
@@ -313,16 +299,16 @@ export const ApproveTransactionScreen: FC<ApproveTransactionScreenProps> = ({
         !useLedger
           ? "Sign"
           : ledgerState === undefined
-          ? "Sign with Ledger"
-          : (ledgerState === "detecting") || (ledgerState === "notfound")
-          ? "Ledger: Detecting"
-          : ledgerState === "signing"
-          ? "Ledger: Signing"
-          : ledgerState === "succeeded"
-          ? "Ledger: Succeeded"
-          : ledgerState === "failed"
-          ? "Ledger: Failed"
-          : ledgerState
+            ? "Sign with Ledger"
+            : (ledgerState === "detecting") || (ledgerState === "notfound")
+              ? "Ledger: Detecting"
+              : ledgerState === "signing"
+                ? "Ledger: Signing"
+                : ledgerState === "succeeded"
+                  ? "Ledger: Succeeded"
+                  : ledgerState === "failed"
+                    ? "Ledger: Failed"
+                    : ledgerState
```

Comment:
Does Argent-X has a format command to enforce the same coding style?

Reply:
Sorry, it should be my editor config, I have created an [issue](https://github.com/alephium/extension-wallet/issues/146) to introduce style check Workflow.